plugin/evm/config.go

```diff
@@ -90,9 +94,13 @@ type Config struct {
 	KeystoreInsecureUnlockAllowed bool   `json:"keystore-insecure-unlock-allowed"`
 
 	// Gossip Settings
-	RemoteTxGossipOnlyEnabled bool     `json:"remote-tx-gossip-only-enabled"`
-	TxRegossipFrequency       Duration `json:"tx-regossip-frequency"`
-	TxRegossipMaxSize         int      `json:"tx-regossip-max-size"`
+	RemoteGossipOnlyEnabled       bool             `json:"remote-gossip-only-enabled"`
+	RegossipFrequency             Duration         `json:"regossip-frequency"`
+	RegossipMaxTxs                int              `json:"regossip-max-size"`
```

Comment:
should we avoid dropping the `tx-` prefix for backwards config compatibility for these 3 flags?

Reply:
I think it is fine...no one currently uses these